### ISSUE TITLE
[macOS] Ozone: Font size is too small on 4K display, no option to change it

### ISSUE THREAD

**Issue by firelad97** [libretro/RetroArch#9063]
## Description

On unchangeable 4K resolution display on my MacBook Pro, the font appears very small, small with no option to increase the size using Ozone menu driver. I've searched online but to no avail.

XMB menu driver has menu scale factor, which is also a suitable option for Ozone. It really needs one.

### Expected behavior

Ozone menu driver having the option to change the font size (or menu scale factor) so it will no longer appear too small on 4K.

### Actual behavior

On launch:
https://i.imgur.com/srC2yEN.png

Settings > User Interface > Appearance:
https://i.imgur.com/8LQ8A9q.png

Just having a MacBook or a 4K monitor sit by a few inches from you will make it very, very difficult to read on.

### Steps to reproduce the bug

1. Open RetroArch on macOS with a 4K display.
2. Go to Settings > Drivers > Menu and change it to Ozone.
3. Restart RetroArch.

### Bisect Results

Unrelated.

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.7.7, build date May 6 2019
(Git version is showing "GIT_VERSION")

It was downloaded from the official website. Metal2 version.

### Environment information

- OS: macOS Mojave/Catalina
- Compiler: Clang/LLVM (10.0.0 (clang-1000.11.45.5)) 64-bit

**Comment by SaulGoodman1337** [comment]
any news here? i have exactly the same problem

**Comment by Clappingfetus** [comment]
Same issue here as well.

**Comment by escalade** [comment]
I need this too :)

**Comment by fmaida** [comment]
I do not own a 4K monitor but I would use this feature a lot on my Fire Stick 4K connected to a tv set. I actually switched the menu driver back to XMB exactly because the font size is too small to read for me on Ozone at the moment.

**Comment by itsumimario** [comment]
I have the same problem with my 4K monitor. Essentially the “menu_scale_factor” parameter in the config file is not respected by Ozone (yet it is respected by XMB, allowing scaling of the UI to appropriate human readable size). 

This makes Ozone nearly unusable on 4K monitors. Ozone in its current state is accessibility unfriendly for vision impaired-people.

**Comment by ChristenEdmundson** [comment]
This problem is also present on Android, where screens are small but high density. I don't believe the ozone theme has scaling on any platform.

**Comment by zoltanvb** [comment]
Solved by #10111, more scaling improvements have been submitted since.

**Comment by domdfcoding** [comment]
I'm not sure it is solved. On 1.170 the scale option only has an effect up to 1.43x, and beyond that it never gets any bigger. On Linux but a small handheld screen.

**Comment by sonninnos** [comment]
Yes, Ozone limits the scale depending on screen width so that it does not become unusable with the sidebar stuff and whatnot. Sure perhaps it can be adjusted, but it must exist in any case.
